app/controllers/api/v0/reactions_controller.rb

```diff
@@ -1,25 +1,24 @@
 module Api
   module V0
-    class ReactionsController < ApplicationController
+    class ReactionsController < ApiController
       skip_before_action :verify_authenticity_token
 
       def create
-        @user = valid_user
-
-        unless @user
-          render json: { message: "invalid_user" }, status: :unprocessable_entity
+        reaction_user = load_reaction_user
```

Comment:
I was confused because I thought it was the logged in user, but it's actually the reaction's user